### DISCUSSION TITLE
Buglab讓AI自己debug

### DISCUSSION THREAD

**Discussion by charlottechuang** [1101-GS4719-C/Python-Community-GS4719-C#318] [General]
微軟提出的一個模型是可以自己debug
看起來似乎很夢幻(不用整天找bug找得要死) 但好像很多人對於會不會修復一個bug又冒出另一個bug這件事感到很困惑就是了🥲🥲
https://www.techbang.com/posts/92908-thanos-coming-microsoft-released-buglab-no-need-to-label-gan?from=home_news&fbclid=IwAR3PX86xEpbzhPpD4jf5iNKdXAVqy8lrvRjf5pKPQufM1PsJBBdbiyNeysM

**Comment by ChinChun0220** [comment]
所以有bug出來我要先思考是自動找bug出問題，還是真的bug，XD

**Comment by cody910120** [comment]
如果有一個資工系朋友就解決一切了

**Comment by mesakiyuta** [comment]
感覺離實用還有一段距離，但蠻期待可以成功，這樣以後就不用跟程式碼乾瞪眼了
